tools/build.py

```diff
@@ -114,61 +129,60 @@ def _BuildProject(self, download_helper_object, project_definition):
 
     return True
 
-  def _BuildProjectForDistribution(
-      self, build_helper_object, source_helper_object, distribution):
-    """Builds a project for a specific distribution.
+  def CheckBuildDependencies(self, project_definition):
+    """Checks if the build dependencies a project are met.
```

Comment:
missing are word from this docstring - "for a project"

Reply:
This hasn't been addressed.

Reply:
ack, likely missed this due to github UI